Makefile

```diff
@@ -65,9 +66,17 @@ deps:
 
 ## install: Install all build binaries into the $PREFIX (/usr/local/ by default) directory.
```

Comment:
[nit] this line is still correct but it could be made more general:

```suggestion
## install: Install the celestia-node binary.
```

and then a comment could be added above `install-global` to say:

```make
## install-global: Install the celestia-node binary into the $PREFIX (/usr/local/ by default) directory.
install-global:
```